test-api.html

```diff
@@ -6,8 +6,153 @@
   <script src="dist/main.js"></script>
 </head>
 <body>
-  <script>
+<script>
+  
+    // First, we're going to create the Todo List type, so that users can create
+    // todo lists.
+  
+    MCT.type('example.todo', new MCT.Type({
```

Comment:
This seems a bit repetitive.  Could just be `MCT.type({key: 'blah', metadata: {..} ...})`.  Not sure why we need to construct types and then register them.